Xamarin.Forms.Core/Items/CarouselView.cs

```diff
@@ -18,6 +18,14 @@ public class CarouselView : ItemsView
 		public const string PreviousItemVisualState = "PreviousItem";
 		public const string DefaultItemVisualState = "DefaultItem";
 
+		public static readonly BindableProperty LoopProperty = BindableProperty.Create(nameof(Loop), typeof(bool), typeof(CarouselView), false, BindingMode.OneTime);
```

Comment:
I think the default should be true.
```suggestion
		public static readonly BindableProperty LoopProperty = BindableProperty.Create(nameof(Loop), typeof(bool), typeof(CarouselView), true, BindingMode.OneTime);
```